sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_storage.py

```diff
@@ -101,6 +110,9 @@ def __init__(
         self._lease_period = lease_period
         self._maintenance_task.daemon = True
         self._maintenance_task.start()
+        self._enabled = self._check_and_set_folder_permissions()
+        if not self._enabled:
+            logger.error("Could not set secure permissions on storage folder.")
```

Comment:
`logger.warning`

Reply:
We do log errors whenever a feature was not able to be set, like AAD, in this case retry logic will not be enabled and data could be missing